android/assets/screens/credits.gdxui

```diff
@@ -89,6 +89,9 @@
                 <Label>Chinese:</Label>
                 <Label>Lu Xu</Label>
                 <Label/>
+                <Label>PortugueseBR:</Label>
```

Comment:
The text here is shown in the credits screen:
![screenshot-2022-09-10-100711 256](https://user-images.githubusercontent.com/3575/189474910-fc9f118e-7b44-4f68-9646-b4e8e217ffb2.png)

It should say something like "Brazilian Portuguese".